README.md

```diff
@@ -18,10 +18,14 @@ Ruby 2.0 or higher is required as of version 1.6
     gem install right_api_client
 
 ## Versioning
-The right\_api\_client gem is versioned using the usual X.Y.Z notation, where X.Y is the
-RightScale API version, and Z is the client version. For example, if you want to use
-RightScale API 1.5, you should use the latest version of the 1.6 gem. This will ensure
-that you get the latest bug fixes for the client that is compatible with that API version.
+* version 1.6.x
+  - api 1.5
+  - ruby 2.0 and above
+  - patch level recieves improvements and bug fixes moving forward.
+* version 1.5.x
+  - api 1.5
+  - ruby 1.8, 1.9.3
+  - patch level receives only security or high priority fixes if requested.
```

Comment:
I think we should use some more capital letters, especially for things like API and Ruby.